src/contrib/cluster/Akka.Cluster.Sharding/ShardedDaemonProcess.cs

```diff
@@ -78,7 +78,9 @@ public MessageExtractor(int maxNumberOfShards)
         }
 
         public override string? EntityId(object message) => (message as ShardingEnvelope)?.EntityId;
-        public override object? EntityMessage(object message) => (message as ShardingEnvelope)?.Message;
+        
+        // Due to https://github.com/akkadotnet/akka.net/issues/7470 we will want to return the underlying content too
+        public override object? EntityMessage(object message) => (message as ShardingEnvelope)?.Message ?? message;
```

Comment:
This PR would have broken this - which is making me wonder why https://getakka.net/articles/debugging/rules/AK2001.html didn't catch this. Any ideas @Arkatufus ?